libraries/apollo-server/package.json

```diff
@@ -19,6 +19,6 @@
     "typescript": "4.2.4"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.2.0"
```

Comment:
why do we even need this in these generic `libaries` projects?

Reply:
Probably a mistake, I'll remove them